concordium-consensus/src/Concordium/KonsensusV1/Types.hs

```diff
@@ -182,6 +186,27 @@ finalizerByIndex :: FinalizationCommittee -> FinalizerIndex -> Maybe FinalizerIn
 finalizerByIndex finCom finInd =
     committeeFinalizers finCom Vector.!? fromIntegral (theFinalizerIndex finInd)
 
+-- | Hash of the finalization committee, only the weight and BLS verify key of each finalizer are used
+-- for computing this.
+newtype FinalizationCommitteeHash = FinalizationCommitteeHash
+    { theFinalizationCommitteeHash :: Hash.Hash
+    }
+
+-- | Compute the hash of the finalization committee. Only the weight and BLS verify key of each
+-- finalizer are used for computing this.
+computeFinalizationCommitteeHash :: FinalizationCommittee -> FinalizationCommitteeHash
```

Comment:
I think a unit test of this would be nice. Maybe even one that shows that some validator with a certain key and weight is in the committee?

Reply:
I imagine this would fit better as a test using the merkle proofs introduce in another PR by Thomas.